### ISSUE TITLE
Several usability issues for non-technical users

### ISSUE THREAD

**Issue by rillig** [EFForg/privacybadgerchrome#314]
I’m a German software developer, and I am evaluating whether I should recommend Privacy Badger to my family so that they can surf privately. The main requirement is that everything they see is understandable, written in German and that it works as expected. During the installation from scratch, I found multiple issues, which are grouped here together since they form a story.

* When I search for "privacy badger" in the German Google, the first search hit is https://www.eff.org/de/node/73969. The "de" in the URL looks to me as if it means "German version of node 73969", but the page is in English.

* The "canonical" of https://www.eff.org/de/node/73969 refers back to itself instead of the more human-friendly URL https://www.eff.org/privacybadger.

* On the page https://www.eff.org/de/node/73969, when I click into the address bar of the browser and then press Tab, a tongue saying "Skip to main content" is displayed. When I click on that, "#main-content" appears in the address bar, but I’m still at the top of the page.

* The text of the big green Install button is so pale that it looks like it were disabled, but it isn’t.

* The page https://www.eff.org/node/73969 is in English, although my browser prefers German.

* After installing the extension, the box "Privacy Badger has been added to Chrome" appears, but only for half a second. This may be because I visited the installer page in incognito mode. You probably cannot do anything about this, but I wanted to mention it, since it’s part of the story.

* The firstRun.html page is in English, which my parents still don’t understand.

* The page firstRun.html contains too much technical babble. It should be completely rewritten by someone who doesn’t know anything about the internet. The technical babble should still be made available, but only behind a "technical details for experts" link.

* The page firstRun.html has quite long lines, so the line-height should be 135% to improve readability.

* It’s not the extension that has three states, but each domain.

* The text "Privacy Badger" in the main heading links to https://www.eff.org/ instead of a page that is specific to Privacy Badger.

* When I start surfing, the Privacy Badger icon shows the number 17 on red background. This means to me that there are 17 "red" issues. But this is not the case.

* So I hover the mouse over the icon, and the tooltip says only "Privacy Badger". No information about the 17, though. I had expected to get a summary like "12 domains blocked, 3 cookies blocked, 2 domains ok".

* To get more details I click on the icon. Now I get a list of 17 green lines. This does not correspond to the number in the icon, which has a _red_ background.

* The small question mark after "These sliders let you control how privacy badger handles each tracker." looks like a question mark, not like a help icon. It looks to me as if Privacy Badger were asking me something.
 
* When I click the small question mark, I am led to the help page, to the specific answer. The question to that answer is not visible, but it should be, so I understand the context without having to scroll up.

Additionally, I have the following questions:

* Does Privacy Badger respect the incognito mode and does not save anything to permanent storage in that mode? I cannot check this myself easily, since Privacy Badger doesn’t show me a list of blocked and cookie-blocked domains.

* Privacy Badger always talks about blocking _domains_, but the UI suggests that it blocks only _hostnames_. Is that correct?

In summary, these are too many negative points, so I cannot yet recommend it to non-technical people.

**Comment by cooperq** [comment]
@rillig thank you for your comments. Privacy badger is definitely in need of some UI love and I will definitely take all this into consideration when I get around to doing UI work. In the meantime please feel free to submit patches to fix any UI problems that are bothering you and I will be happy to merge them.

**Comment by pde** [comment]
Mark, can your team triage some of these website issues and fix at least the easy ones?

**Comment by pde** [comment]
Perhaps this is now for jeremy...

**Comment by maxateff** [comment]
@pde @cooperq Mark's working on internationalization for eff.org/privacybadger.  His work on that is blocked until we get confirmation of a freeze on the page's content, but we're ready to help whenever we reach that point.

**Comment by cooperq** [comment]
Cool, we should be good on the page content after wednesday, I will let you guys know.

**Comment by ghostwords** [comment]
Hi @rillig, it's been a while! I think all of your feedback should have gotten addressed by now. It would be great if you could give Privacy Badger another look, and maybe once again let us know of any issues you notice.

To answer your previous question about Incognito: Privacy Badger does not learn in Incognito by default (in order to avoid recording any Incognito site/tracker domains in Privacy Badger's storage). You can choose to enable learning in Incognito, however.

**Comment by rillig** [comment]
Wow, I'm impressed. Both installing and using Privacy Badger works really smoothly now. Many thanks to everyone who was involved in improving all these small details, which were really many.
